trie/triedb/pathdb/database.go

```diff
@@ -395,26 +417,31 @@ func (db *Database) Recover(root common.Hash, loader triestate.TrieLoader) error
 		start = time.Now()
 		dl    = db.tree.bottom()
 	)
+	log.Info("recover info", "root", root.String(), "state id", dl.stateID())
```

Comment:
Refine this log to describe the execution context better. eg "recover info" -> "Recover info"; "root" -> "target_root"; "state id"->"current_disklayer_state_id".

Reply:
Test log is deleted.